client/src/components/home/modules-section.tsx

```diff
@@ -0,0 +1,335 @@
+          colorClass={bgColorClass}
+          intro={intro}
+          slug={slug}
+        >
+          <div className={cn('flex h-40 w-[245px] flex-col gap-4 p-6', bgColorClass)}>{intro}</div>
+        </HoverTooltip>
+      </DialogTrigger>
+      <DialogContent
+        onCloseAutoFocus={(e) => e.preventDefault()}
+        contentClassName="p-0"
+        className="!w-[792px] !max-w-[792px]"
+      >
+        <div className="flex h-full w-full justify-between">
+          <div className="h-full w-1/2 flex-1 space-y-6 p-10">
+            <Icon className={cn('currentColor h-fit w-[48px] min-w-[48px]', textColorClass)} />
+            <div className="font-serif text-2xl leading-10">{title}</div>
+            <div className="flex flex-col gap-4">{modulesInfo[slug].modalText}</div>
+          </div>
+          <div className="flex h-full w-1/2 flex-1 border-l border-dashed border-gray-700 border-opacity-50 p-10 pt-[160px]">
+            <ul className="flex flex-col gap-4">
+              {modulesInfo[slug].modalList.map((listElement) => (
+                <li key={listElement} className="flex gap-2 text-sm">
+                  <CheckCircle />
+                  {listElement}
+                </li>
+              ))}
+            </ul>
+          </div>
+        </div>
+      </DialogContent>
+    </Dialog>
+  );
+};
+
+const ModulesSection = () => (
+  <div id="modules" className="scroll-mt-[100px]">
+    <div className="mb-[120px] flex items-center justify-center">
+      <div className="flex flex-col items-center gap-6">
+        <div className="text-center font-serif text-sm uppercase leading-[14px] tracking-wider text-gray-400">
+          5 modules for climate impact
+        </div>
+        <div className="flex flex-col items-center gap-8">
+          <div className="font-serif text-[32px] font-semibold text-gray-700">
+            What can Impact4Soil do for me?
+          </div>
+          <div className="h-2 w-20 bg-teal-500" />
+          <div className="w-[600px] text-center text-gray-700">
+            Impact4Soil is a unique, reliable and inter-institutional knowledge platform that brings
+            altogether actors and experts of the soil carbon community.
+            <br />
+            It provides key information on the latest researches and their results, including
+            methodologies for soil carbon balance monitoring, practices, networks, and data.
+          </div>
+        </div>
+      </div>
+    </div>
+    <div className="relative flex min-h-[616px] items-center justify-center">
+      <div className="absolute top-0">
+        <img src="/images/map.png" className="h-[616px] w-full" alt="" />
+      </div>
+      <div className="absolute top-0 z-10 flex h-[616px] flex-col justify-center">
+        <div className="flex gap-24">
+          <Card
+            title="Geospatial Data"
+            slug="geospatial-data"
+            Icon={Globe2}
+            bgColorClass="bg-yellow-500"
```

Comment:
The colours of the cards are dimmer on the design.

Reply:
Yes, I talked about that with Lidia. They needed to be the same as the real modules so we are going to use that colors and dont make them change on hover